packages/sdk/src/utils/__test__/helpers.test.ts

```diff
@@ -30,16 +38,50 @@ describe('constructDepositDataEvmSubstrate', () => {
     const expectedDepositData = '0x1234';
 
     jest.spyOn(utils, 'parseUnits').mockReturnValueOnce(BigNumber.from('100000000000000000000'));
-    jest.spyOn(utils, 'isAddress').mockReturnValueOnce(true);
-    jest.spyOn(utils, 'arrayify').mockReturnValueOnce(new Uint8Array([18, 52, 86]));
+    jest
+      .spyOn(helpers, 'getRecipientAddressInBytes')
+      .mockReturnValueOnce(new Uint8Array([18, 52, 86]));
     jest.spyOn(utils, 'hexlify').mockReturnValueOnce(expectedDepositData);
 
-    const depositData = constructDepositDataEvmSubstrate(tokenAmount, recipientAddress, decimals);
+    const depositData = createERCDepositData(tokenAmount, recipientAddress, decimals);
 
     expect(utils.parseUnits).toHaveBeenCalledWith(tokenAmount, decimals);
-    expect(utils.isAddress).toHaveBeenCalledWith(recipientAddress);
-    expect(utils.arrayify).toHaveBeenCalledWith(recipientAddress);
+    expect(helpers.getRecipientAddressInBytes).toHaveBeenCalledWith(recipientAddress);
     expect(utils.hexlify).toHaveBeenCalledWith(expectedBytesArr);
     expect(depositData).toEqual(expectedDepositData);
   });
 });
+
+describe('getRecipientAddressInBytes', () => {
+  it('should convert an EVM address to a Uint8Array of bytes', () => {
```

Comment:
`getRecipientAddressInBytes(evmAddress)` basically does `utils.arrayify(evmAddress)` so if the test is called convert to `Uint8Array` it should be something like:
```
result = getRecipientAddressInBytes(evmAddress)
expect(result).isinstanceof(Uint8Array).to.be.true;
```

Reply:
added `expect(result).toBeInstanceOf(Uint8Array);`